src/sentry/dynamic_sampling/prioritise_projects.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import time
+from collections import defaultdict
+from datetime import datetime, timedelta
+from typing import Mapping, Sequence
+
+from snuba_sdk import (
+    Column,
+    Condition,
+    Direction,
+    Entity,
+    Function,
+    Granularity,
+    Op,
+    OrderBy,
+    Query,
+    Request,
+)
+
+from sentry.sentry_metrics.indexer.strings import TRANSACTION_METRICS_NAMES
+from sentry.snuba.dataset import Dataset, EntityKey
+from sentry.snuba.metrics.naming_layer.mri import TransactionMRI
+from sentry.utils.snuba import raw_snql_query
+
+logger = logging.getLogger(__name__)
+MAX_SECONDS = 60
+CHUNK_SIZE = 1000
+
+
+def fetch_projects_with_total_volumes() -> Mapping[int, Sequence[Sequence[int]]]:
```

Comment:
Please document what the ints and sequences are. From the existing comment I'm assuming that the outer `int` is the organization ID, but it's not clear what the inner one looks like. 

In some modules we already started to introduce types for `ProjectId` and `OrganizationId` (e.g. in sentry.release_health). This could help with interpreting the response structure.

Would it be even better if instead of `Sequence[int]` we use `tuple[ProjectId, int]`?

Reply:
done in 6965588358